plat/rockchip/rk3368/include/platform_def.h

```diff
@@ -74,6 +74,8 @@
 					 PLATFORM_CLUSTER_COUNT +	\
 					 PLATFORM_CORE_COUNT)
 
+#define PLATFORM_CLST_TO_CPUID_SHIFT	6
```

Comment:
Prefixing this macro with `PLATFORM_` and placing it in `platform_def.h` implies that this is a platform porting macro when it isn't. Better to just call it `CLST_TO_CPUID_SHIFT` and include in `rk3399_def.h`.